contracts/gateway/GatewayBase.sol

```diff
@@ -338,20 +338,71 @@ contract GatewayBase is Organized {
         emit BountyChangeConfirmed(previousBountyAmount_, changedBountyAmount_);
     }
 
+    /**
+     * @notice Method to get the active message hash and its status from inbox
+     *         for the given account address.
+     *
+     * @dev If message hash do not exists for the given account address then
```

Comment:
```suggestion
     * @dev If message hash does not exist for the given account address then
```
Also, is this not relevant to the user of the function? Why in `@dev`?

Reply:
Removed from the @dev and added to notice.